packages/amesos/CMakeLists.txt

```diff
@@ -35,6 +35,15 @@ IF(NOT TPL_ENABLE_MPI)
 ENDIF()
 ENDIF()
 
+# if using CSC, must be parallel and needs Pardiso_MKL
```

Comment:
@iyamazaki is this set up so that `-DAmesos2_ENABLE_CSS_MKL` is automagically enabled only if `-DTPL_ENABLE_CSS_MKL=ON` is set? And can you confirm that if you enable MKL and Pardiso, but have `-DTPL_ENABLE_CSS_MKL=OFF`, that  `HAVE_AMESOS_CSS_MKL` is undefined/OFF?
I'm checking to prevent issues where an MKL version might not have CSS available (I'm thinking of the Albany issue)

Reply:
@ndellingwood. We only require ``MPI``, and ``Pardiso_mkl`` is not needed. I also had a serial build with ``TPL_ENABLE_CSS_MKL``, and in that case, it will just turn off CSS.